SDKBuildScripts/initejs.bat

```diff
@@ -0,0 +1,3 @@
+pushd ..
+npm install ejs
```

Comment:
In order for the npm install to work you'll need a _package.json_ file defining the minimum ejs version we'll be using, and ideally a _package-lock.json_ file (which will be autogenerated) that will define what's the version we'll keep using moving forward. They both should exist on the root folder. 

Example of a valid _package.json_ file which will install any version of EJS greater than 2.5.7:

```
{
  "dependencies": {
    "ejs": "^2.5.7"
  }
}
```

Or... maybe I'm missing something on the file structure 😅 Is the _initejs.bat_  scipt working locally for you right now?

Reply:
Added